### DISCUSSION TITLE
Stand alone controller REM dropping generic device settings

### DISCUSSION THREAD

**Discussion by craywm** [tagyoureit/nodejs-poolController#1007] [REM]
Hello all!
I recently replaced my pump motor (Superpump Century single speed) with the VGreen EVO 165 motor and had trouble integrating RS-485 communication with the Century Connect Automation within the pump. I was able to establish reliable communication with the motor and integration into my Nixie schedules, but during the troubleshooting process, I may have inadvertently changed a setting that is now causing some issue with REM. 

I deleted the REM generic devices during the troubleshooting process, and reset them up (temp sensors / RPi temp / Pressure transducer) and re-initialized the I2C bus. I confirmed everything was functioning with Nixie appropriately as well as new motor (no surprises, I ended up using an external USB since motor baud rate is not 9600). After a power cycle, I noticed REM is no longer feeding temp data to Dashboard and upon checking REM, my I2C bus and all generic devices are again gone/not setup. 

Its been a while since I set this up new and I'm sure I'm overlooking some setting. Can someone help me identify why my REM configuration is being dropped after every power cycle? I'd prefer not to setup and recalibrate all my relays after every power cycle! Let me know if a replay would be useful for troubleshooting. I thought maybe it was an issue with PM2, but I couldn't find any indication that PM2 would have changed the config. 

Configuration: Nixie Single Body
Sequent Boards: 8-relays hat & MEGA-BAS
RS-485 Comms: 8-relays hat (IC40) & USB Dongle (EVO VS Motor); I had a previous issue with my MEGA-BAS RS-485 pass-through so I use the 8-relay instead. 
Auto start: PM2 running
Sensors: 1 temp sensor (air) and 1 pressure transducer
![image](https://github.com/user-attachments/assets/8dc40ba8-ea7f-4920-bc76-c39dfe87405e)
![image](https://github.com/user-attachments/assets/8c0013d7-e603-4cae-b550-b81a70ba50bd)

**Comment by rstrouse** [comment]
Upload your controllerConfig.json file from the data directory in REM.  I want to see if it is missing the I2c data.

**Reply by craywm** [reply]
I did a power cycle yesterday and the controller seemed to retain the I2C bus 
[controllerConfig.json](https://github.com/user-attachments/files/16350618/controllerConfig.json)
settings. I'd still like to get to the bottom of the issue to make sure my settings are correct. Attached is the Config file.
